desc/compute/_curve.py

```diff
@@ -644,43 +726,24 @@ def _x_sss_FourierXYZCurve(params, transforms, profiles, data, **kwargs):
     dim=3,
     params=["X", "Y", "Z", "knots", "rotmat", "shift"],
     transforms={
-        "method": [],
+        "intervals": [],
     },
     profiles=[],
     coordinates="s",
     data=["s"],
     parameterization="desc.geometry.curve.SplineXYZCurve",
     basis="{'rpz', 'xyz'}: Basis for returned vectors, Default 'rpz'",
+    method="{'cubic', 'linear'}: Interpolation type, Default 'cubic'",
```

Comment:
I think there are a lot more than just cubic or linear right? there is a list on the SplineXYZCurve docstring. We could add all the options here (or just say refer to SplineXYZCurve for the full list or something), we should be accurate here since this description will end up [in the docs](https://desc-docs.readthedocs.io/en/latest/variables.html#id15)

Reply:
Yeah there's a lot more, just forgot to add them. Okay gotcha

Reply:
I just said to look at the SplineXYZCurve docstring, but i can also add this link: https://desc-docs.readthedocs.io/en/latest/_api/geometry/desc.geometry.SplineXYZCurve.html (just not sure how to add without going over character limit)